### PR TITLE
docs: describe options for stack offset

### PR DESCRIPTION
**Pull request by domoritz**

### CODE REVIEW

File: src/transform.ts [hunk 1]
```diff
@@ -124,7 +124,9 @@ export interface StackTransform {
    */
   groupby: FieldName[];
   /**
-   * Mode for stacking marks.
+   * Mode for stacking marks. One of `"zero"` (default), `"center"`, or `"normalize"`.
```
Comment (kanitw) [inline src/transform.ts]: Missing description for zero. 

We have description for stack in encoding that you might want to copy from
https://vega.github.io/vega-lite/docs/stack.html#encoding

Comment (domoritz) [reply src/transform.ts]: I copied the description from Vega.

Comment (domoritz) [reply src/transform.ts]: Done

Comment (kanitw) [reply src/transform.ts]: yeah, but given this will appear on the same page as the encoding one. Please revise to see if they look consistent

**Pull request merged**